website_multi_company_demo/tests/test_domain_updating.py

```diff
@@ -11,4 +11,5 @@ def test_authenticate(self):
             base_location="http://build-123.runbot.example.com",
         )
         uid = self.registry['res.users'].authenticate(db_name, 'admin', 'admin', env)
-        self.assertEqual(uid, 1)
+        # since Odoo 12.0 uid == 1 == SUPERUSER_ID
```

Comment:
You shall comment the code, not the changes you just did
```suggestion
        # since Odoo 12.0 admin user is 2
```

Reply:
Also, add a test to check that redefined method updates website domains. See https://github.com/it-projects-llc/website-addons/pull/214/files#diff-134e6cc8f473b5dd1f8b7b0bfae3e3a8R20